fastlane/Fastfile

```diff
@@ -127,6 +129,31 @@ platform :ios do
         build_and_upload_app(options)
     end
 
+    lane :trigger_api_build_cirrus_ci do |options|
+        authorization_header = { "Authorization" => "Bearer #{ENV['CIRRUS_CI_API_TOKEN']}" }
+        uri = 'https://api.cirrus-ci.com/graphql'
+        create_build = HTTParty.post(
+            uri,
+            body: "{ \"query\": \"mutation CreateBuildMutation($input: RepositoryCreateBuildInput!) { createBuild(input: $input) { build { id } } }\", \"variables\": { \"input\": {\"repositoryId\": \"4507403593449472\", \"branch\": \"#{ENV['GITHUB_REF_NAME']}\", \"clientMutationId\": \"trigger-build-on-cirrus\"}} }",
```

Comment:
can we use multiline string and avoid escaping the `"`?
```suggestion
            body: <<-BODY
"{ \"query\": \"mutation CreateBuildMutation($input: RepositoryCreateBuildInput!) { createBuild(input: $input) { build { id } } }\", \"variables\": { \"input\": {\"repositoryId\": \"4507403593449472\", \"branch\": \"#{ENV['GITHUB_REF_NAME']}\", \"clientMutationId\": \"trigger-build-on-cirrus\"}} }"
BODY
```

Reply:
Please recheck